storage/remote/storage.go

```diff
@@ -65,12 +65,16 @@ type Storage struct {
 
 // NewStorage returns a remote.Storage.
 func NewStorage(l *slog.Logger, reg prometheus.Registerer, stCallback startTimeCallback, walDir string, flushDeadline time.Duration, sm ReadyScrapeManager) *Storage {
+	// register remote storage metrics in	custom registry
```

Comment:
```suggestion
	// Register remote storage metrics in custom registry.
```

Reply:
But maybe this comment is not needed at all. It just spells out what the code is clearly doing. No reason to explain the code here.